conjure-java-core/src/main/java/com/palantir/conjure/java/types/EndpointErrorGenerator.java

```diff
@@ -165,7 +165,6 @@ private static List<MethodSpec> generateConditionalExceptionFactories(
                                     options,
                                     exceptionClassName,
                                     Optional.empty())
-                            .addException(exceptionClassName)
```

Comment:
> The undertow interfaces are still generated with a throws clause

how does this relate to this statement? I was expecting to still see `throws` on methods in the generated integration input but it looks like they disappeared.